src/store/reducers/mainContactReducer.js~HEAD

```diff
@@ -0,0 +1,25 @@
+import { ACTION_TYPES } from '../actionTypes';
```

Comment:
this file shouldn't be here right ?

Reply:
I don't understand why this is here also, it's from my previous PR. 
But I think it got some how pushed before closing my last pr.
Anyways, it has been modified & merged already...